### ISSUE TITLE
Auto-scroll to bottom when mention is used in composer

### ISSUE THREAD

**Issue by VolodLytvynenko** [status-im/status-react#21710]
### Problem:
When a mention is added to the composer, the user is automatically scrolled to the bottom of the composer.

### Steps to Reproduce:
1. Open the composer in a chat or text input field.
2. Type a message and insert a mention in the upper part of composer

### Actual Result:
The auto-scrolls occur to the bottom of the composer when a mention is added 

https://github.com/user-attachments/assets/23e0191b-9cf8-4562-85f6-bd796ba25b75

### Expected Result:
The user's position in the composer should remain unchanged

### ENV:
Nightly 28, 2024

**Comment by ajayesivan** [comment]
It looks like the issue is related to `setNativeProps`. We're updating the composer `TextInput` value using `setNativeProps` within the mention logic.

https://github.com/status-im/status-mobile/blob/f27c569c9d5fe6119ce71f5b5116681e9936c7c0/src/status_im/contexts/chat/messenger/composer/events.cljs#L229-L232

**Comment by ajayesivan** [comment]
Are we using `setNativeProps` for performance reasons? Any objections to using a controlled input instead?

cc: @flexsurfer / @Parveshdhull

**Comment by flexsurfer** [comment]
yes setNativeProps is used for performance reasons, controlled input shouldn't be used
